applications/GeoMechanicsApplication/custom_elements/small_strain_U_Pw_diff_order_element.cpp

```diff
@@ -173,6 +173,31 @@ Vector SmallStrainUPwDiffOrderElement::GetPressures(const size_t n_nodes) const
     return pressure;
 }
 
+void set_arithmetic_average_pressure(Geometry<Node>&                               rGeometry,
+                                     const Vector&                                 rPressure,
+                                     const std::vector<std::pair<size_t, size_t>>& index_pairs,
+                                     size_t                                        dest_offset = 0)
```

Comment:
To align with the style guide (+ a bit more clarity for the last argument)
```suggestion
void SetArithmeticAveragePressure(Geometry<Node>&                                   rGeometry,
                                     const Vector&                                 rPressure,
                                     const std::vector<std::pair<size_t, size_t>>& rIndexPairs,
                                     size_t                                        DestinationOffset= 0)
```
Same holds for the other function

Reply:
changed